src/core/plugins/oas3/components/servers.jsx

```diff
@@ -134,7 +134,7 @@ export default class Servers extends React.Component {
                       <td>{name}</td>
                       <td>
                         { val.get("enum") ?
-                          <select data-variable={name} onChange={this.onServerVariableValueChange}>
+                          <select aria-label="server variables" data-variable={name} onChange={this.onServerVariableValueChange}>
```

Comment:
can we do 
```suggestion
                          <select aria-label={`server variable ${name}`} data-variable={name} onChange={this.onServerVariableValueChange}>
```
so that multiple rows get a unique and more descriptive label?